test/functional/p2p_mutated_blocks.py

```
@@ -0,0 +1,96 @@
+from test_framework.blocktools import (
+    COINBASE_MATURITY,
+    create_block,
+    add_witness_commitment,
+    NORMAL_GBT_REQUEST_PARAMS,
+)
+from test_framework.util import assert_equal
+from test_framework.wallet import MiniWallet
+import copy
+
+class MutatedBlocksTest(BitcoinTestFramework):
+    def set_test_params(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `set_test_params`

Add return type annotation: `None`

(ANN201)